src_rebuild/premake5.lua

```diff
@@ -46,21 +46,24 @@ project "PSX"
 
     defines { GAME_REGION }
 
-    files { 
+    files {
         "EMULATOR/**.h", 
+        "EMULATOR/**.H", 
         "EMULATOR/**.c", 
+        "EMULATOR/**.C", 
         "EMULATOR/**.cpp",
+        "EMULATOR/**.CPP",
     }
 
     defines { "OGL", "GLEW" }
 
-    includedirs { 
-        SDL2_DIR.."/include",
-        GLEW_DIR.."/include",
-        OPENAL_DIR.."/include",
-    }
-
     filter "system:Windows"
+        includedirs { 
```

Comment:
I would not move it to only this systen, SDL, OpenAL-soft and GLEW can be custom built.